care/facility/tasks/redis_index.py

```diff
@@ -4,6 +4,7 @@
 from celery import shared_task
 from celery.utils.log import get_task_logger
 from django.core.cache import cache
+from django.core.management import BaseCommand
```

Comment:
```suggestion
```

Reply:
Since we're using cache, I've followed your recommendations and deleted the import to use cache